src/components/myPlantDetails/overViewMyPlantDetails.tsx

```diff
@@ -1,83 +1,59 @@
+import {
+  ArrowUpFromLine,
+  ArrowUpToLine,
+  BetweenVerticalEnd,
+  Thermometer,
+  Tractor,
+  TreeDeciduous,
+  Utensils,
+  UtensilsCrossed,
+  BarChart4,
+} from "lucide-react";
 import { Plant } from "../../models/plant/plant";
+import Detailsquare from "./detailSquare";
 
 interface OverviewMyPlantDetailsProps {
   myPlant: Plant;
 }
 
 const OverviewMyPlantDetails = ({ myPlant }: OverviewMyPlantDetailsProps) => {
+  const createCelsiusFormat = (maxTemp: number, minTemp: number): string => {
+    return `${minTemp} - ${maxTemp}`;
```

Comment:
Nu zul je dus bijvoorbeeld _15 - 23°C_ zien. Misschien is het beter als het _15°C - 23°C_ is. Maar ben dit zelf niet zeker, wat ik het beste vind. Wat denk jij?

Reply:
15°C - 23°C  vind ik ook beter ;)